src/pages/SearchHelpPage.vue

```diff
@@ -0,0 +1,167 @@
+<template>
+<div class="page grid-container full">
+  <header-section showNavSearch="true"></header-section>
+  <div class="page_body">
+    <h1>CC Search Advanced Search User Guide</h1>
+    <p>
+      When using CC Search, users can search for things like
+      <a href="https://ccsearch.creativecommons.org/search?q=cats">"cats"</a> and
+      <a href="https://ccsearch.creativecommons.org/search?q=dogs">"dogs"</a>.
+      Users can filter the search query for
+      <a href="https://search.creativecommons.org/search?q=dogs&provider=&li=&lt=commercial&searchBy">
+      photos of "dogs" that I can use commercially
+      </a>.
+      In addition to that, it also supports using a special search syntax that lets users apply
+      those filters directly on the search input box, without having to use the provided filter
+      user interface. The following guide explains how you can take advantage of that special
+      search syntax to enjoy a more powerful search capability.
+    </p>
+
+    <h2>Search by title</h2>
+    <p>
+      You can search by title by putting "<em>title:</em>" in front of the query.
+    </p>
+    <p>Example:
+    <a href="https://search.creativecommons.org/search?q=title%3A%20cats">
+      <em>title: cats</em>
+    </a>
+    </p>
+
+    <h2>Search by license</h2>
+    <p>
+      You can search by license by putting "<em>license:</em>" in front of the query.
+    </p>
+    <p>Example:
+    <a href='https://search.creativecommons.org/search?q=license%3A%20"by"'>
+      <em>license: "by"</em>
```

Comment:
this doesn't look like it's working, I'm not getting any search results for this query on CC Search.